src/vm/comutilnative.cpp

```diff
@@ -892,12 +892,28 @@ FCIMPL5(void, GCInterface::GetMemoryInfo, UINT32* highMemLoadThreshold, UINT64*
 
     FC_GC_POLL_NOT_NEEDED();
     
-    return GCHeapUtilities::GetGCHeap()->GetMemoryInfo(highMemLoadThreshold, totalPhysicalMem, 
+    return GCHeapUtilities::GetGCHeap()->GetMemoryInfo(highMemLoadThreshold, totalPhysicalMem,
                                                        lastRecordedMemLoad, 
                                                        lastRecordedHeapSize, lastRecordedFragmentation);
 }
 FCIMPLEND
 
+void QCALLTYPE GCInterface::GetGCConfigValue(const LPCWSTR key, QCall::StringHandleOnStack result)
+{
+    QCALL_CONTRACT;
+
+    BEGIN_QCALL;
+
+    const char* value = GCHeapUtilities::GetGCHeap()->GetGCConfigValue(key);
+    if (value != nullptr)
+    {
+        result.Set(value);
+        delete value;
```

Comment:
This will leak memory when `result.Set` throws. Use Holder - we use holders everywhere else in the runtime.